README.md

```diff
@@ -164,6 +168,21 @@ For more information on different components that compose our commit messages, p
 
 Additionally, CI will use `commitlint` to validate the commits associated with a PR in the `Lint and Validate` job.
 
+### Adding Documentation
+
+*This does not apply to visual component packages.*
+
+However, if your package generates **types**, then add a `build:docs` script to your `package.json` file similar to the one in [@kong-ui-public/analytics-utilities](https://github.com/Kong/public-ui-components/blob/main/packages/analytics/analytics-utilties/package.json#L12)
+
+Then append `build:docs` to the existing `build` script in your package's `package.json`.  
+
+```json
+"scripts": {
+  "build": "run-s typecheck build:package build:types build:docs",
```

Comment:
I don't believe our CI will check in these files on build -- this likely needs to be a manual process before committing changes to a PR.